handlers/svg-open-street-map/osm-svg.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2022 IMAGE Project, Shared Reality Lab, McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# and our Additional Terms along with this program.
+# If not, see
+# <https://github.com/Shared-Reality-Lab/IMAGE-server/blob/main/LICENSE>.
+
+from flask import Flask, jsonify, request
+import json
+import jsonschema
+from jsonschema.exceptions import ValidationError
+import logging
+import time
+import drawSvg as draw
+from colour import Color
+app = Flask(__name__)
+
+
+@app.route("/handler", methods=["POST"])
+def handle():
+    logging.debug("Received request")
+    # Load necessary schema files
+    with open("./schemas/definitions.json") as f:
+        definitions_schema = json.load(f)
+    with open("./schemas/request.schema.json") as f:
+        request_schema = json.load(f)
+    with open("./schemas/handler-response.schema.json") as f:
+        response_schema = json.load(f)
+    with open("./schemas/renderers/svglayers.schema.json") as f:
+        renderer_schema = json.load(f)
+    store = {
+        definitions_schema["$id"]: definitions_schema,
+        request_schema["$id"]: request_schema,
+        response_schema["$id"]: response_schema,
+        renderer_schema["$id"]: renderer_schema,
+    }
+    resolver = jsonschema.RefResolver.from_schema(
+        request_schema, store=store
+    )
+    # Get and validate the request contents
+    contents = request.get_json()
+    try:
+        validator = jsonschema.Draft7Validator(
+            request_schema, resolver=resolver
+        )
+        validator.validate(contents)
+    except ValidationError as e:
+        logging.error(e)
+        return jsonify("Invalid request received!"), 400
+
+    preprocessor = contents['preprocessors']
```

Comment:
May be `None` - the `preprocessors` field is not required in a request body, but if it is provided it has to follow specifications.